products/vuetify.md

```diff
@@ -57,6 +59,7 @@ releases:
     eol: 2020-07-31
     latest: "1.5.24"
     lts: 2019-07-31
+    eoes: true
```

Comment:
```suggestion
```

Was there commercial support for vuetify 1 in the past ? If not this should be removed.